src/components/TodoItem.jsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
```

Comment:
- [ ] I suggest enabling linters by removing  this comment

Reply:
Thanks, And I will do it.